fullscreen.bs

```diff
@@ -340,23 +330,26 @@ attribute's getter must run these steps:
   <p>As part of the next <a>animation frame task</a>, run these substeps:
 
   <ol>
+   <li><p>If <var>doc</var>'s <a>fullscreen element</a> is null, then resolve <var>promise</var>
+   with undefined and terminate these steps.
 
    <li><p>Let <var>exitDocs</var> be the result of
    <a lt="collect documents to unfullscreen">collecting documents to unfullscreen</a> given
-   <var>doc</var>.
+   <var>doc</var> and the <var>fully flag</var>.
    <!-- cross-process -->
 
-   <li><p>If <var>resize</var> is true and <var>topLevelDoc</var> is not in <var>exitDocs</var>,
-   <a>fully exit fullscreen</a> <var>topLevelDoc</var>, reject <var>promise</var> with a
-   <code>TypeError</code> exception, and terminate these steps.
-
    <li><p>If <var>exitDocs</var> is the empty set, append <var>doc</var> to <var>exitDocs</var>.
 
    <li><p>If <var>exitDocs</var>'s last <a for=/>document</a> has a
    <a>browsing context container</a>, append that <a>browsing context container</a>'s
    <a>node document</a> to <var>exitDocs</var>.
    <!-- cross-process -->
 
+   <li><p>If <var>resize</var> is true, then: while <var>exitDocs</var>'s last <a for=/>document</a>
+   has a <a>browsing context container</a>, append that <a>browsing context container</a>'s
+   <a>node document</a> to <var>exitDocs</var>.
```

Comment:
This would not work, because we only call *unfullscreen an element* on documents in *exitDocs*.